modular_azurepeak/code/modules/reagents/reagent_containers/rotcure.dm

```diff
@@ -7,7 +7,7 @@
 	description = "A putrid substance that appears to be in perpetual motion. It smells and looks of living-sludge."
 	color = "#034212"
 	taste_description = "living sludge"
-	overdose_threshold = 10	//Stops people from downing too much of it.
+	overdose_threshold = 21	//Stops people from downing too much of it.
```

Comment:
what is this arbitrary number
a phial is `volume = 27` so if you drink a full phial you'll overdose but you need twenty to cure it
cure threshold should be `25` and overdose should be `30`

Reply:
also undocumented change lmao